06-Modelling-Rigid-Bodies/02-Running-Rigid-Body-Simulations.ipynb

```diff
@@ -0,0 +1,674 @@
+{
```

Comment:
<p>I think it would be useful to give a short description of the constraint object here/an intuitive picture of what adding this constraint does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>